openedx/adg/lms/applications/admin.py

```diff
@@ -325,6 +335,22 @@ def _save_application_review_info(self, application, request, note):
 
         application.save()
 
+    def _send_acceptance_wait_list_email(self, application, request):
```

Comment:
I agree, but that should be done separately, because we will be changing the main send email method of MandrillClient and we would be changing at the places where we are sending emails. Also what would be the new flow, is not very clear, we will be using the same class method or again, we would have to write helper in client and use that.